docs/get-started/quickstarts/detect-people.md

```diff
@@ -10,16 +10,18 @@ images: ["/icons/components/camera.svg"]
 imageAlt: "Machine Learning"
 ---
 
-Follow this guide to detect people with a webcam on the Viam platform, using a [pre trained ML model from the registry](https://app.viam.com/ml-model/ml-models-scuttle/people):
+{{<youtube embed_url="https://www.youtube-nocookie.com/embed/wci21VRYyWY">}}
```

Comment:
I think this may be a buffering issue you're seeing or maybe youtube trying to pick the right resolution. I am not seeing this (not on incognito either)